create-message/src/main/java/no/unit/nva/publication/messages/create/CreateMessageHandler.java

```diff
@@ -87,10 +87,12 @@ private Publication fetchExistingPublication(CreateMessageRequest input) throws
         }
     }
 
-    private SortableIdentifier sendMessage(CreateMessageRequest input, UserInstance sender, Publication publication,
-                                           UserInstance owner) throws TransactionFailedException, BadRequestException {
+    private SortableIdentifier sendMessage(CreateMessageRequest input,
+                                           UserInstance sender,
+                                           Publication publication
+    ) throws TransactionFailedException, BadRequestException {
         try {
-            return messageService.createMessage(sender, owner, publication.getIdentifier(), input.getMessage());
+            return messageService.createMessage(sender, publication, input.getMessage());
```

Comment:
sending in the whole publication, because we need more data from the publication object (title,owner,publisher)